phase5/SampleLibrary/InvertedIndex.cs

```diff
@@ -0,0 +1,33 @@
+namespace SampleLibrary;
+
+public class InvertedIndex : IInvertedIndex
+{
+
+    public Dictionary<string, List<string>> WordToDocumentMap { get; } = new ();
```

Comment:
It is good to have a document ID to content map so we have access to the content of each document.

Reply:
we do have a map in another part of the code
the inverted index itself does not need to have access to that map